drive/modules/listener/listener.bal

```diff
@@ -27,12 +27,34 @@ import ballerina/time;
 # + domainVerificationFileContent - File content of HTML file used in domain verification.
 # + callbackURL - Callback URL registered.  
 # + clientConfiguration - Drive client connecter configuration.
+# + retryConfiguration - Retry configuration.
+@display{label: "Listener Config"}
 public type ListenerConfiguration record {
+    @display{label: "Port"}
     int port;
+    @display{label: "Callback URL"}
     string callbackURL;
+    @display{label: "Domain Verification File Content"}
     string domainVerificationFileContent;
     Configuration clientConfiguration;
+    @display{label: "Specific Folder ID"}
     string? specificFolderOrFileId = ();
+    RetryConfiguration retryConfiguration?;
+};
+
+# Retry configuration
+#
+# + retryCount - Maximum number of reties allowed.
+# + retryInterval - Interval time between retries.  
+# + initialWaitTime - Initial wait time for domain verification.  
```

Comment:
Initial wait time for domain verification check in seconds (default: z)

Reply:
Sure. Let's change accordingly.